website/static/js/ads.js

```diff
@@ -0,0 +1,6 @@
+/* http://stackoverflow.com/questions/4869154/how-to-detect-adblock-on-my-website
+ * Most adblocks will see that a file with ads in the title is attempting to be
+ * imported and block it. In this case the variable won't exist, which means we will
+ * enter the if.
+*/
+var canRunAds = true;
```

Comment:
This shouldn't be here. This was the problem you were having with branches? You'll want to get this cleared up. It will be a bit of a pain to do, I'm afraid.